src/lib/engine/guesstimate.js

```diff
@@ -82,3 +82,15 @@ function _inputMetricsWithValues(guesstimate: Guesstimate, dGraph: DGraph): Obje
   })
   return [inputs, _.uniq(errors)]
 }
+
+export function inputsFromExpressions(metrics) {
```

Comment:
I'd probably rename this inputsFromExpressionsFn or inputsFromExpressionsFunction or something to indicate it returns a function, really didn't expect that.  Maybe a comment on the top.